src/p_mobj.h

```diff
@@ -240,6 +240,7 @@ enum {
   MIF_LINEDONE = 4,     // Object has activated W1 or S1 linedef via DEH frame
   // mbf21
   MIF_SCROLLING = 8,    // Object is affected by scroller / pusher / puller
+  MIF_FLIP = 10,
```

Comment:
`10`?

Reply:
Oops, fixed in https://github.com/fabiangreffrath/woof/pull/910/commits/b14a2fea19e4d947971cdf7421c7fe5ec7d90f7b